bioblend/_tests/test_util.py

```diff
@@ -60,8 +60,17 @@ def skip_unless_galaxy(min_release=None):
             galaxy_user = galaxy_user_email.split("@", 1)[0]
             galaxy_password = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(20))
 
-            # Create a new user and get a new API key for her
-            new_user = gi.users.create_local_user(galaxy_user, galaxy_user_email, galaxy_password)
+            try:
+                config = gi.config.get_config()
+            except Exception:
+                # If older Galaxy for instance just assume use_remote_user is False.
+                config = {}
+
+            if config.get("use_remote_user", False):
+                new_user = gi.users.create_remote_user(galaxy_user_email)
+            else:
+                # Create a new user and get a new API key for her
```

Comment:
You can move `galaxy_user` and `galaxy_password` definitions (lines 60-61) here.